controller/menu.rb

```diff
@@ -0,0 +1,46 @@
+require 'observer'
```

Comment:
- [ ] Good job with this, now according to requirements, it would be excellent if you use this code (the menu options) inside main.rb, this to be more according to the guidelines to this project, please use the menu options inside `main.rb` 🐶 
![image](https://user-images.githubusercontent.com/20563252/136871189-f7251d78-bb3d-46d5-8ce9-474c0fb7621e.png)

Reply:
Helo reviewer! I tried to go one step ahead, and using the `single responsibility principle` the menu works on its own and it is inside the app class wrapped in its class.
![image](https://user-images.githubusercontent.com/28903404/136871539-47d304d3-b695-4f1d-a0fa-b65ea1f48ab6.png)

You can see I implemented the observer pattern and the menu is observable ( as it should be). The menu generates the options dynamically and can be easily expanded to do more actions, putting it inside the app will remove all of this functionality and will force me to use static references or switch cases.
![image](https://user-images.githubusercontent.com/28903404/136871799-a3d3b526-0784-4c90-b9c2-5b7b5606879e.png)

Regards! and thanks for the feedback!